src/fuzzfetch/fetch.py

```diff
@@ -24,6 +24,7 @@
 import configparser  # pylint: disable=wrong-import-order
 import requests
 
+from datetime import datetime
```

Comment:
let me pylint that for you:

    C: 27, 0: standard import "from datetime import datetime" should be placed before "import requests" (wrong-import-order)